PageObjectPattern/src/test/java/api/stepdefinitions/GetInfoStepDefinitions.java

```diff
@@ -0,0 +1,56 @@
+//done
+package api.stepdefinitions;
+
+import static org.junit.Assert.assertEquals;
+import static org.hamcrest.Matchers.equalTo;
+
+import apiSerenityStep.AppAuthStep;
+import apiSerenityStep.BaseStep;
+import apiSerenityStep.SpotifyApiStep;
+import io.cucumber.java.en.*;
+import io.restassured.response.ValidatableResponse;
+import net.thucydides.core.annotations.Steps;
+
+public class GetInfoStepDefinitions {
+    @Steps
+    AppAuthStep auth;
+
+    @Steps
+    BaseStep baseStep;
+
+    @Steps
+    SpotifyApiStep apiStep;
+
+    @Given("User has app access token")
+    public void getAccessAppToken() {
+        System.out.println("Getting app access token");
+        String newToken = auth.getAppToken();
+        System.out.println("IT IS A NEW TOKEN" + newToken);
+    }
+
+    @When("User sends request to {word}")
+    public void sendGetRequest(String endpoint) {
```

Comment:
If it's a common method for all GET requests, this method might be extracted to CommonStepDefinitions class as well